nodes/test/conftest.py

```
@@ -98,35 +101,30 @@ async def server_available():
-        )
+        pytest.skip(f"Server not available at {TEST_CONFIG.uri}. Run 'builder nodes:test' to start server automatically.")
     return True
 
 
 # =============================================================================
 # Client Fixtures
 # =============================================================================
 
+
 @pytest_asyncio.fixture
 async def client(server_available):
```

Ruff (0.15.10)

[warning] Unused function argument: `server_available`

(ARG001)